src/components/macroview/rightPanel/editForms/MousePressForm.tsx

```diff
@@ -178,16 +250,28 @@ export default function MousePressForm({
               Duration (ms)
             </Text>
           </GridItem>
-          <GridItem w="full">
+          <VStack w="full">
             <Input
               type="number"
+              placeholder={String(DefaultMouseDelay)}
               variant="brandAccent"
               value={mousepressDuration}
               onChange={onMousepressDurationChange}
               onBlur={onInputBlur}
-              isInvalid={Number.isNaN(parseInt(mousepressDuration))}
+              isInvalid={Number.isNaN(mousepressDuration)}
             />
-          </GridItem>
+            <Button
+              variant="brandTertiary"
+              leftIcon={<ResetDefaultIcon />}
+              w="full"
+              value=""
+              m={1}
+              size={['sm', 'md']}
+              onClick={onResetClick}
+            >
+              <Text fontSize={['md', 'md', 'sm']}>Reset to Default</Text>
```

Comment:
The responsive fontSize here is kinda weird? It gets smaller and has a duplicated size?

Reply:
Yeah it is.. tweaked it a bit here: [e9594d9](https://github.com/WootingKb/wooting-macros/pull/194/commits/e9594d9bf58c5fa27625f4549ae3f6d0d4f55abb)